benchmarks/avaHelixChannelentTest/avaHelixChannel_com1DFACfg.ini

```diff
@@ -11,10 +11,6 @@ RunCint = runBasic
 inputf = shp
 # True if full output shall be printed to terminal
 flagOut = False
-# True if entrainment areas shall be included
-flagEnt = True
```

Comment:
where are the flagNoEnt=False?

Reply:
HelixChannel has entrainment

Reply:
yes so since you want the entrainment run you do not need the flagNoEnt no?